dashboard/public/js/main.js

```diff
@@ -103,3 +103,66 @@ function firstClientVisit() {
 		return true;
 	}
 }
+var textBox = document.getElementById("password-box");
+var password = document.getElementById('password');
+
+function passwordStrength() {
+	var parentDiv = document.createElement("div");
+	parentDiv.id = "passtrength-div";
+	
+	var meter = document.createElement("METER");
+	meter.id = "password-strength-meter";
```

Comment:
Don't add extra code which is not required. I think this ID does not have any use in the code.